snuba/datasets/transactions.py

```diff
@@ -83,11 +88,26 @@ def __init__(self):
             write_schema=schema,
         )
 
-        super(TransactionsDataset, self).__init__(
+        super().__init__(
             dataset_schemas=dataset_schemas,
             processor=TransactionsMessageProcessor(),
             default_topic="events",
+            time_group_columns={
+                'start_ts': 'start_ts',
+                'end_ts': 'end_ts',
```

Comment:
Hm… I thought these names weren't supposed to match? I think this might cause issues (but I'm admittedly not quite sure how, so maybe I'm wrong.)

Reply:
This puzzled me a bit, I tried to understand why they did not match on events but did not find a good reason. Let me dig a little deeper into that.